app/src/main/res/values/styles_dark.xml

```diff
@@ -73,7 +73,6 @@
         <item name="toc_scroller_button_color">@color/base18</item>
         <item name="button_background_color">@color/base14</item>
         <item name="tab_placeholder_text_color">@color/base10</item>
-        <item name="thumbnail_image_placeholder">@drawable/ic_image_gray_dark_24dp</item>
```

Comment:
Please also remove the unused resource

Reply:
Sorry about the misunderstanding. "Delete" unused resource would be a proper word.